packages/p2p/src/pages/orders/order-table/__tests__/order-table.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import OrderTable from '../order-table';
+import userEvent from '@testing-library/user-event';
+import { useStores } from 'Stores';
```

Comment:
could ya reorder these

```suggestion
import React from 'react';
import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import { useStores } from 'Stores';
import OrderTable from '../order-table';
```